src/handler.jl

```diff
@@ -23,17 +23,19 @@ Names with a '.' separators will be used to form subdirectories
 (e.g., "Foo.bar.x" will be saved to "\$prefix/Foo/bar/x.jlso").
 """
 function path(handler::Handler{P}, name::String; tags...) where P
-    # Build up a path prefix based on the tags passed in.
-    prefix = Vector{String}(undef, length(tags))
-    for (i, t) in enumerate(tags)
-        prefix[i] = string(first(t), "=", last(t))
-    end
+    with_tags(tags...) do
```

Comment:
the tags passed to `checkpoint` are now treated as the innermost context, which is quite nice

Reply:
nice